source/Octopus.Manager.Tentacle/DeleteWizard/Views/DeleteWelcome.xaml

```diff
@@ -30,7 +29,7 @@
 
                 <StackPanel Margin="10,0,10,0">
                     <TextBlock Style="{StaticResource TextBlock.Big}" TextWrapping="Wrap">
-                    <Run Text="{Binding Path=ApplicationName, StringFormat='Are you sure you want to delete your {0} instance?'}" />
+                    <Run Text="{Binding Mode=OneWay, Path=ApplicationName, StringFormat='Are you sure you want to delete your {0} instance?'}" />
```

Comment:
I have no idea what this means :)

Reply:
default Binding option is "TwoWay", which means changes in UI are synced with the code behind it and vice versa. For this view, we have some read-only properties in the code behind, which may confuse the default binding option. Hence, it is set to "OneWay".